src/CSV_Parser.php

```diff
@@ -363,4 +425,17 @@ private function _makeRows() {
 		$this->setRows($rows);
 		return $this;
 	}
+
+	/**
+	 * Validate character encoding
+	 *
+	 * @return boolean
+	 */
+	private function _isValidEncoding($encoding = null) {
+	    if (in_array($encoding, mb_list_encodings())) {
```

Comment:
This method would just return `in_array()` as this would return a boolean by default anyway. Ref: http://php.net/manual/en/function.in-array.php

i.e.
```php
return in_array($encoding, mb_list_encodings());
```